library/logging.po

```diff
@@ -810,13 +831,20 @@ msgid ""
 "func:`shutdown` is called. Subclasses should ensure that this gets called "
 "from overridden :meth:`close` methods."
 msgstr ""
+"Recycle toutes les ressources utilisées par le gestionnaire. Cette version "
+"ne fait rien de particulier mais elle retire le gestionnaire de la liste "
```

Comment:
« ne fait rien de particulier mais  » me paraît un peu étrange. Mais en fait, j'ai aussi du mal à comprendre l'original : que veut dire « to close a list » ? Est-ce que ce sont en fait les éléments de la liste qui sont fermés en tant que descripteurs de fichier ? Merci d'éclairer ma lanterne :-)

Reply:
https://github.com/python/cpython/blob/main/Lib/logging/__init__.py#L992

Reply:
Oui ce n'est pas la liste qui est « closée » mais le gestionnaire dans liste par l'appel à _shutdown_.
Cette fonction ne fait que supprimer l'entrée dans la liste. Je mets « rien de particulier »  pour indiquer que des sous-classes de _Handler_ peuvent avoir leur logique « particulière » ici.

Reply:
OK, merci à vous deux.